pyproject.toml

```
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "supaero2023"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.8"
+gepetuto = "^1.0.0"
+crocoddyl = "^2.0.1"
+example-robot-data = "^4.0.7"
+virtualenv ="^20.24.2"
+notebook ="^6.0.3"
+matplotlib = "^3.7.2"
+meshcat = "^0.3.2"
+quadprog = "^0.1.11"
+scipy = "^1.10.1"
+tensorflow = "^2.13.0"
```

```suggestion
gepetuto = "^1.0.0"
crocoddyl = "^2.0.1"
matplotlib = "^3.7.2"
meshcat = "^0.3.2"
quadprog = "^0.1.11"
tensorflow = "^2.13.0"
```

I think the other ones are dependencies of those ones